sshuttle/client.py

```diff
@@ -582,16 +600,23 @@ def udp_connection_is_allowed(dstip, dstport, srcip):
     if _excluded_sources and srcip in _excluded_sources and (_excluded_sources[srcip] / 1000.0) >= ctime:
         debug1("Connection from a source excluded from the ACL\n")
         return True
-    
-    if not _allowed_sources:
-        debug3("Connection not allowed - allowed sources exception - not _allowed_sources\n")
-        return False
-    if (srcip not in _allowed_sources):
-        debug3("Connection not allowed - allowed sources exception - (srcip not in _allowed_sources)\n")
-        return False
-    if (srcip in _allowed_sources and (_allowed_sources[srcip] / 1000.0) < ctime):
-        debug3("Connection not allowed - allowed sources exception - (srcip in _allowed_sources and (_allowed_sources[srcip] / 1000.0) < ctime)\n")
-        return False
+
+    check_allowed_sources = True
+
+    if _always_connected == ALWAYS_CONNECTED_ON:
+        debug1("UDP connection source is allowed because alwaysConnected mode is ON\n")
```

Comment:
add a comment indicating that this is one of the places where we'll need to check the workspace IP when having the 'alwaysConnected' flag on the workspace instead of just the room.

Reply:
debug1 seems very verbose, should be debug3 level

Reply:
can do